### PR TITLE
Add Japanese translation

### PR DESCRIPTION
**Pull request by aiotter**
Add Japanese translation along with translator credit at README.md.
I inserted some css modification at the top of the index.html so that `<em>` tag should be seen more naturally in Japanese.

I'm not sure the title is appropriate.
It's not the direct translation of 'Don't ask to ask, just ask', but just as short as 'Don't ask to ask'.
I gave priority to catchy phrase, simplicity and shortness.

Can you review my translation, @atsuoishimoto and @Lazialize?

### CODE REVIEW

**Review by Lazialize — APPROVED** [review]

File: ja/index.html [hunk 1]
```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+  <title>「質問していいですか」と聞かないで</title>
```
Comment (atsuoishimoto) [inline ja/index.html:4]: nit: 今のままでも良いと思いますが、タイトル部分で原文のリズムの良さが感じられないのがちょっと寂しい気もします。ちょっとふざけ気味だけど


- 聞いていいですかと聞かないで。さっさと聞いて。


とか…

File: ja/index.html [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    </p>
+    <blockquote>
+      <span class="name">Hogehoge123:</span>
+      <p class="message">
+        Java 分かる人いますか？
+      </p>
+    </blockquote>
+    <p>
+      これはいろいろな意味で良くない聞き方です．
+      その人が<em>本当に</em>聞きたいことは実際はこのような感じでしょう．
```
Comment (atsuoishimoto) [inline ja/index.html:38]: この質問は、<em>実際には<em>こんなことを言っているのです。

File: ja/index.html [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    </p>
+    <blockquote>
+      <span class="name">Hogehoge123:</span>
+      <p class="message">
+        Java 分かる人で，もしかしたら実際のところ Java と関係ないかもしれないし，Java を全然知らない人でも答えられるかもしれない私の問題を調べるのに専念するつもりがある人はいますか？
+      </p>
+    </blockquote>
+    <p>
+      知識のある人がこのような質問を認めようとしない理由はたくさんあります．
```
Comment (atsuoishimoto) [inline ja/index.html:47]: Admitのニュアンスは消えちゃいますが、「このような質問に答えない理由は~」でよいかと。

File: ja/index.html [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+      このような質問をすることによって，あなたは自分が聞こうとしていると思っていることよりもはるかに多くのことを要求しているのです．
```
Comment (atsuoishimoto) [inline ja/index.html:48]: nit: ちょっと直訳調なので、「この質問は、あなたが自分で思っているよりはるかに多くのことを要求しているのです」とか。

File: ja/index.html (hunk 3 above)
Comment (atsuoishimoto) [reply ja/index.html:47]: 「本当に知識のある人々は、こういう質問には答えようとしません。」でもいいか

**Comment by aiotter** [comment]
Thanks @Lazialize @atsuoishimoto!
I'll take a couple of days to think about better translation of the title. If there is no better translation coming into my mind, the current title will be the final draft.

**Comment by daima3629** [comment]
It's just my opinion, but I think it would be better to replace "," with "、" and "." with "。".

File: ja/index.html [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+<body>
+  <main>
+    <h1>「質問していいですか」と聞かないで</h1>
+    <p>
+      私がよく見ているオンラインのチャットルームでは，ときどき誰かが入ってきてこのようなことを言います．
+    </p>
+    <blockquote>
+      <span class="name">Hogehoge123:</span>
+      <p class="message">
+        Java 分かる人いますか？
```
Comment (sevenc-nanashi) [inline ja/index.html:33]:
```suggestion
        Java分かる人いますか？
```
変に空白開けない方がチャット感が出ると思いました。（個人の意見です）

File: ja/index.html [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+      </p>
+    </blockquote>
+    <p>
+      これはいろいろな意味で良くない聞き方です．
+      この質問は，<em>実際には</em>こんなことを言っているのです．
+    </p>
+    <blockquote>
+      <span class="name">Hogehoge123:</span>
+      <p class="message">
+        Java 分かる人で，もしかしたら実際のところ Java と関係ないかもしれないし，Java を全然知らない人でも答えられるかもしれない私の問題を調べるのに専念するつもりがある人はいますか？
```
Comment (sevenc-nanashi) [inline ja/index.html:43]:
```suggestion
        Java分かる人で，もしかしたら実際のところJavaと関係ないかもしれないし，Javaを全然知らない人でも答えられるかもしれない私の問題を調べるのに専念するつもりがある人はいますか？
```
同上。カンマ関連は他の人に任せます。

File: ja/index.html [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+      また，他の人を不必要に排除していることにもなります．
+      私は一度も使ったことがない言語やライブラリに関する質問によく回答しますが，これはその答えが（プログラマ的には）常識的なものだからです．
+    </p>
+    <p>
+      あるいは，その質問はこのように見られてしまうかもしれません．
+    </p>
+    <blockquote>
+      <span class="name">Hogehoge123:</span>
+      <p class="message">
+        私には Java に関する質問がありますが，答えられそうな人がいなそうなら面倒くさいのでちゃんと言葉にしたくないです
```
Comment (sevenc-nanashi) [inline ja/index.html:62]:
```suggestion
        私にはJavaに関する質問がありますが，答えられそうな人がいなそうなら面倒くさいのでちゃんと言葉にしたくないです
```
同上。

File: ja/index.html [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+      これはただ怠け者なだけです．
+      あなたが自分の問題を解決するために努力したくないのなら，なぜ私たちが努力しなくてはいけないのですか？
+    </p>
+    <p>
+      解決方法は，質問してもいいか聞く代わりに，ただ質問をすることです．
+      そのチャンネルを開いておいて時々ちらっと様子を見るだけの人は，あなたの「質問してもいいですか」の質問にはなかなか答えてくれませんが，あなたが実際の問題の説明をすれば興味を持って答えてくれるかもしれません．
+    </p>
+    <p>
+      つまり，まとめると，
+      <em>「Java 分かる人いますか？」</em>
```
Comment (sevenc-nanashi) [inline ja/index.html:75]:
```suggestion
      <em>「Java分かる人いますか？」</em>
```
同上。

File: ja/index.html [hunk 9]
```diff
@@ -0,0 +1,94 @@
...
+      と聞くのではなく，
+      <em>「(問題)を Java で解決するにはどうすればいいですか？　関係しそうなことは(関連情報)です」</em>
```
Comment (sevenc-nanashi) [inline ja/index.html:77]:
```suggestion
      <em>「(問題)をJavaで解決するにはどうすればいいですか？　関係しそうなことは(関連情報)です」</em>
```

**Review by sevenc-nanashi — COMMENTED** [review]
Just my opinion, I think chat message has no space around `Java` , so I removed. it's just my opinion, feel free to ignore this.

**Comment by aiotter** [comment]
@sevenc-nanashi thx for your comment.
You suggested editing inside `<blockquote>` tag. Everything inside this tag is rendered in monospace font. FYR: https://github.com/maunium/dontasktoask.com/blob/009f7ccbaa5506f2aaaf21c1735128cf3a0df6eb/style.css#L60-L70
Without space between Japanese and English character, it seems ugly and a bit hard to read.
So I won't commit your suggestion. Thanks again anyway!

**Comment by aiotter** [comment]
With and without space on monospace font:
image

**Comment by aiotter** [comment]
@daima3629 According to Japanese Industrial Standards, H.4.2 of [JIS Z 8301:2019](http://kikakurui.com/z8/Z8301-2019-01.html), punctuation characters should be '，' and '。'.

> H.4.2.1 一般
> 区切り符号には，句点“。”，読点としてのコンマ“，”，中点“・”及びコロン“：”を用いる。斜線“／”は，通常，“及び／又は”などを表記する場合に用いる。なお，セミコロン“；”は，用いない。

I will follow this rule. The current punctuation character '．' will be substituted with '。'.
I appreciate your comment!

**Comment by aiotter** [comment]
6352266 <- wow, the short commit hash with only numerical characters! I've never seen this happened😲

**Review by sevenc-nanashi — APPROVED** [review]

**Comment by aiotter** [comment]
Okay, let's go with this draft.
I don't think the current title is the best translation, but I don't have better idea.
Maybe someone with better idea will create an improving PR someday.

@tulir Can you review this PR?
I think we did our best for the Japanese translation. Hope it will be merged!

File: ja/index.html (hunk 1 above)
Comment (sevenc-nanashi) [reply ja/index.html:4]: > 聞いて良いか聞くな、とっと聞け。

とかどうでしょうか（ちょっと口調があれだけど）。

**Review by tulir — APPROVED** [review]
Thanks!

**Pull request merged**